crates/python/tests/qcs_config/settings.toml

```diff
@@ -3,8 +3,8 @@ default_profile_name = "default"
 [auth_servers]
 
   [auth_servers.default]
-    client_id = ""
-    issuer = "https://rigetticomputing.okta.com/oauth2/ausmoidj5qavd4v310h7"
+    client_id = "0oa3ykoirzDKpkfzk357"
+    issuer = "https://auth.qcs.rigetti.com/oauth2/aus8jcovzG0gW2TUG355"
```

Comment:
to match https://github.com/rigetti/qcs-api-client-rust/blob/main/qcs-api-client-common/src/configuration/settings.rs#L157-L159